sqlalchemy_solr/http.py

```diff
@@ -63,6 +65,10 @@ def create_connect_args(self, url, **kwargs):
                 if kwargs["use_ssl"] in [True, "True", "true"]:
                     self.proto = "https://"
 
+            if "token" in url.query:
```

Comment:
Should `url.query` be replaced with `kwargs` in this block?

Reply:
Actually, I am using superset 3.0.0

Reply:
and I have tested that kwargs did not receive the use_ssl or token, that's why I have to use url.query

solr://172.20.20.245:8443/solr/enterpriseSearch?token=xyz&use_ssl=true

Reply:
Yeah, looks like `**kwargs` is no longer used.

Reply:
Thanks for quick merge,I am sending one more PR related to SSL support Fix its a small change